WebJobs.Script.proj

```diff
@@ -107,7 +108,11 @@
              BuildInParallel="$(BuildInParallel)"/>
   </Target>
 
-  <Target Name="PackageScriptHost" DependsOnTargets="Build">
+  <Target Name="DownloadRuntimeNugetClient">
+    <DownloadNuGet OutputFileName="$(FunctionsSiteExtensionPath)\bin\tools\nuget.exe" MinimumVersion="3.3.0" DownloadUrl="https://facavalcspn.blob.core.windows.net/public/nuget.exe"  />
```

Comment:
Yes. It's odd that this is under one I created with my name, but I'm using the storage account used by CI. I can work on updating all of that so it's a dedicated account.